src/App.jsx

```diff
@@ -1,16 +1,24 @@
 import "./App.css";
 import { useState } from "react";
+import { Routes, Route } from "react-router-dom";
 import Header from "./components/Header";
 import Articles from "./components/Articles";
+import OneArticle from "./components/OneArticle";
 
 function App() {
   const [article, setArticle] = useState([]);
 
   return (
-    <div>
+    <>
       <Header />
-      <Articles article={article} setArticle={setArticle} />
-    </div>
+      <Routes>
+        <Route
+          path="/"
+          element={<Articles article={article} setArticle={setArticle} />}
```

Comment:
this comment isn't for ticket 5 but do you need the state for `article` and `setArticle` to be declared in _App.jsx_ or could it be moved into _Articles.jsx_ ? 🤔